plugins/explore/.eslintrc.js

```diff
@@ -1 +1,6 @@
-module.exports = require('@backstage/cli/config/eslint-factory')(__dirname);
+module.exports = require('@backstage/cli/config/eslint-factory')(__dirname, {
+    rules: {
+      '@backstage/no-top-level-material-ui-4-imports': 'error',
+    },
+  });
+  
```

Comment:
Ahh can you set your editor to run prettier on save? :) This catches my eye every single time.